ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/autovpn-edge-no-default-policy.yml

```diff
@@ -53,7 +53,9 @@ wan_router:
 
 wan_path_groups:
   - name: MPLS
-    ipsec: False
+    ipsec:
+      static_peers: false
+      dynamic_peers: false
```

Comment:
no change to autovpn config?

Reply:
The devices don't have mpls path group.

Reply:
yeah makes sense.. :p can we add it to them then? so that we test

Reply:
[0caa614](https://github.com/aristanetworks/avd/pull/3695/commits/0caa6140ce329f464d00a011cfa7ef798a324109)